README.md

```diff
@@ -622,3 +627,28 @@ deploy               Deploy wheel to public s3 bucket
 clean                Clean hidden and compiled files
 help                 Show help message
 ```
+
+## 🔌 How to use production models
+
+We'll go through an example on how to download data and models, and use them for prediction and training using the cli commands.
+
+Data can be downloaded from here after you make an account: http://participants-area.bioasq.org/datasets/
```

Comment:
I think data are not needed for the basic user. My idea is that the basic user is not trying to use the data to train or anything but apply to their own data. They want to know how to download a model and use it to make predictions.